charts/gardener-extension-provider-azure/templates/priorityclass.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: {{ include "priorityclassversion" . }}
+kind: PriorityClass
+metadata:
+  name: gardener-extension-provider-azure
+value: 1000000000
```

Comment:
Hmm, this is the same value, that the gardenlet has. Should we give the extensions less priority than the gardenlet?

Reply:
I think it makes sense to use the same value. The extension are similarly important like the gardenlet.

Reply:
Ok, good :)